builtin/mainmenu/settings/settingtypes.lua

```diff
@@ -103,6 +99,25 @@ local function parse_setting_line(settings, line, read_all, base_level, allow_se
 		return "Tried to add \"secure.\" setting"
 	end
 
+	local requires = {}
+	local last_line = #current_comment > 0 and current_comment[#current_comment]:trim()
+	if last_line and last_line:lower():sub(1, 9) == "requires:" then
```

Comment:
This format should be documented in the first comment of settingtypes.txt, even if it is yet only used for Minetest-internal settings.